adapters/repos/db/lsmkv/bucket_roaring_set_range.go

```diff
@@ -32,3 +43,33 @@ func (b *Bucket) RoaringSetRangeRemove(key uint64, values ...uint64) error {
 
 	return b.active.roaringSetRangeRemove(key, values...)
 }
+
+type ReaderRoaringSetRange interface {
+	Read(ctx context.Context, value uint64, operator filters.Operator) (*sroar.Bitmap, error)
+	Close()
+}
+
+func (b *Bucket) ReaderRoaringSetRange() ReaderRoaringSetRange {
```

Comment:
I'd think the active memtable is not included so to not block insertions, isn't it?

Reply:
active memtable is added in line 65. flushing memtable is added in line 63 (if present).
or you have something else in mind?

Reply:
ah, haven't seen that line when active memtable is added